homeassistant/components/drop/__init__.py

```diff
@@ -0,0 +1,69 @@
+"""The DROP integration."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components import mqtt
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import (
+    CONF_COORDINATOR,
+    CONF_DATA_TOPIC,
+    CONF_DEVICE_TYPE,
+    CONF_UNIQUE_ID,
+    DOMAIN as DROP_DOMAIN,
```

Comment:
I agree the 'DROP_' prefix is unnecessary and have removed it across the board in 1a3b630b474c551054932330a8763dabed553c04.  It was just a pattern I saw in other integration projects as I was getting familiar with HA.

I also agree that changing the `drop` domain to `drop_connect` is reasonable.  Our product line is just called 'DROP', but marketing started using 'DROP Connect' when we couldn't secure drop.com many years ago.  Since then, it does get used for disambiguation from time to time.  I'll make that change after I've worked through the rest of your (appreciated) suggestions.

Reply:
The HA domain for this integration has been changed to `drop_connect` in 2f5517f13effd814a25a4f9854f9a283f487a904.